pkg/skaffold/initializer/prompt/prompt.go

```diff
@@ -78,3 +80,23 @@ confirmLoop:
 	}
 	return false, nil
 }
+
+// PortForwardResource prompts the user to give a port to forward the current resource on
+func portForwardResource(out io.Writer, imageName string) (int, error) {
+	reader := bufio.NewReader(os.Stdin)
+
+	fmt.Fprintf(out, "Select port to forward for %s (leave blank for none): ", imageName)
+
+	response, err := reader.ReadString('\n')
+	if err != nil {
+		return 0, fmt.Errorf("reading user confirmation: %w", err)
+	}
+
+	response = strings.ToLower(strings.TrimSpace(response))
+	if response == "" {
+		return 0, nil
```

Comment:
if user leaves this blank, would the port forward be written with port number "0"?

Reply:
0 gets treated as a special case here. If the port number is zero, then no port-forward information is generated for that image

Reply:
got it. i see the test now.